app/src/main/java/com/hedvig/app/feature/chat/ui/ChatActivity.kt

```diff
@@ -77,7 +79,8 @@ class ChatActivity : BaseActivity(R.layout.activity_chat) {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        tracker.closeChat()
+        trackingFacade.track("click_start_chat")
```

Comment:
Do we need to add `if (savedInstanceState == null) { trackingFacade.track("click_start_chat") }` here to only show this on the first time the activity starts? Wouldn't we get duplicate events on the event of a process death?
I know I am the one that asked for this change but it'd be a shame if we get wrong duplicate events because of me 😅